lightspeed/modules/proc_resolve-ansible-lint-rule-violations.adoc 

```diff
@@ -0,0 +1,53 @@
+
+== How does the {AnsibleContentParser} handle rule violations?
+
+* *Using autofixes*
++
+The {AnsibleContentParser} runs ansible-lint with the `--fix=all` option to perform autofixes, which can fix or simplify fixing issues identified by that rule.
++
+If ansible-lint identifies rule violations that have an associated autofix, it automatically fixes or simplifies the issues that violate the rules. If ansible-lint identifies rule violations that do not have an associated autofix, it reports these instances as rule violations which you must fix manually. 
+For more information about autofixes, see link:https://ansible.readthedocs.io/projects/lint/autofix/[Autofix] in Ansible Lint Documentation. 
+
+* *Using syntax-checks*
++
+Ansible-lint also performs syntax checks while scanning your Ansible files. If any syntax-check errors are found, ansible-lint stops processing the files. 
+For more information about syntax-check errors, link:https://ansible.readthedocs.io/projects/lint/rules/syntax-check/[syntax-check] in Ansible Lint Documentation.
++
+The {AnsibleContentParser} handles syntax-check rule violations in the following manner:
+
+** If `syntax-check` errors are found in the first execution of ansible-lint, {AnsibleContentParser} generates a list of files that contain the rule violations.
+
+** If one or more `syntax-check` errors are found in the first execution of ansible-lint, the {AnsibleContentParser} runs ansible-lint again but excludes the files with syntax-check errors. After the scan is completed, the {AnsibleContentParser} generates a list of files that contain rule violations. The list includes all files that caused syntax-check errors as well as other rule violations.  The {AnsibleContentParser} excludes files with rule violations in all future scans, and the final training data set does not include data from the excluded files.
+
+.Procedure
+Use one of the following methods to resolve ansible-lint rule violations:
+
+* Run the {AnsibleContentParser} with the  `--no-exclude` option
+If any rule violations, including syntax-check errors, are found, the execution is aborted with an error and no training data set is created.
+
+* Limit the set of rules that ansible-lint uses to scan your data with the  `--profile` option
++
+It is recommended that you fix the files with rule violations. However, if you do not want to modify the source files, you can limit the set of rules that ansible-lint uses to scan your data. 
+To limit the set of rules that ansible-lint uses to scan your data, specify the --profile option with a predefined profile (for example, minimum, basic, moderate, safety, shared, or production profiles) or by using ansible-lint configuration files. For more information, see the link:https://ansible.readthedocs.io/projects/lint/[Ansible Lint documentation]. 
```

Comment:
Should "--profile" be surrounded with single quotes (as it is when referenced above)?

Reply:
Yes, and this is done. Thank you.